db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/WindowResultSetNode.java

```diff
@@ -736,6 +736,8 @@ private void addWindowFunctionColumns() throws StandardException {
             List<ValueNode> operands = windowFunctionNode.getOperands();
             int[] inputVIDs = new int[operands.size()];
             int i = 0;
+            ColumnPullupVisitor columnPuller =
+               new ColumnPullupVisitor(WindowFunctionNode.class, LOG, ((SingleChildResultSetNode) this.childResult).getChildResult(), rCtoCRfactory);
```

Comment:
Can you move `((SingleChildResultSetNode) this.childResult).getChildResult()` to a separate local value?